Configuration-Interaction/CI_DL.py

```diff
@@ -117,20 +112,14 @@
 for CI_ITER in range(max_guess - 1):
 
     # Subspace Matrix, Gij = < bi | H | bj >
-    #for i in range(num_vecs - num_eig, num_vecs):
     for i in range(0, num_vecs):
         # Build sigma for each b
         ciwfn.sigma(cvecs, svecs, i, i)
```

Comment:
The `CIWave` module wasnt really built to do the above so a little patience with it is required.